### PR TITLE
Fix aquarium demo

### PR DESCRIPTION
**Pull request by rndexe**
1. Fix AccumulativeShadows and RandomizedLight intensity
2. Fix useMask not working as mentioned in this issue
https://github.com/pmndrs/drei/issues/1949

- [x] ready for merge?

### CODE REVIEW

File: demos/aquarium/src/App.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import turtleModel from './model_52a_-_kemps_ridley_sea_turtle_no_id-transformed
 
 export default function App({ spheres }) {
   return (
-    <Canvas shadows camera={{ position: [30, 0, -3], fov: 35, near: 1, far: 50 }}>
+    <Canvas shadows camera={{ position: [30, 0, -3], fov: 35, near: 1, far: 50 }} gl={{ stencil:true }}>
```
Comment (abernier) [inline demos/aquarium/src/App.jsx:11]: is this `gl={{ stencil:true }}` really necessary in this case? I tried without, seems to have no effect :/

Comment (abernier) [reply demos/aquarium/src/App.jsx:11]: what's default value?

Comment (rndexe) [reply demos/aquarium/src/App.jsx:11]: without stencil: true
Screenshot from 2024-07-31 12-28-02

with stencil:true
Screenshot from 2024-07-31 12-26-49

**Comment by abernier** [comment]
this one should have the CI failed (because this demo has a visual test defined) -- but it didn't and make me realise that the CI still needs tweaks: https://github.com/pmndrs/examples/pull/34

so I keep it open until I've fixed this ;)

**Pull request merged**